app/code/Magento/SendFriendGraphQl/Model/Resolver/ProductService.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\SendFriendGraphQl\Model\Resolver;
+
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\Catalog\Model\Product\Visibility;
+use Magento\Catalog\Api\Data\ProductInterface;
+use Magento\Framework\GraphQl\Exception\GraphQlNoSuchEntityException;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+/**
+ * Class ProductService
+ */
+class ProductService
```

Comment:
@rleshchenko:
1. could you please use the `Command` pattern for this.
Like - class name `GetProduct` and only one method is present called `execute`.

2. Please move to `Provider` directory - for example `\Magento\PaypalGraphQl\Model\Provider\Config` (sorry that I did not mention it earlier)